### PR TITLE
W 012 3

### PR DESCRIPTION
**Pull request by WOJTASK1**
This is beta version of the navigation bar. For now, all of the requirements set at the task are fullfiled.

### CODE REVIEW

File: frontend/src/components/navbar/Navbar.css
Comment (MartynaSzymanskaGitHub) [inline frontend/src/components/navbar/Navbar.css:1]: change look of the navbar to more figma style

File: frontend/src/components/navbar/Navbar.jsx
Comment (MartynaSzymanskaGitHub) [inline frontend/src/components/navbar/Navbar.jsx:1]: code is duplicated and can be shortened . Needs refactoring

**Review by MartynaSzymanskaGitHub — CHANGES_REQUESTED** [review]
needs to change apperance of navbar
and file conflicts must be resolved

**Comment by WOJTASK1** [comment]
outdated.

**Pull request closed without merging**